blockchain/types/tx_internal_data_ethereum_set_code.go

```diff
@@ -68,9 +69,9 @@ type TxInternalDataEthereumSetCode struct {
 	AuthorizationList []SetCodeAuthorization
 
 	// Signature values
-	V *big.Int `gencodec:"required" json:"v"`
```

Comment:
How about using same `uint256.Int` type as geth?

Reply:
Sounds good. I've ignored these commits once in the previous [PR](https://github.com/kaiachain/kaia/pull/216):
* https://github.com/ethereum/go-ethereum/pull/30078/commits/1506e06dca0d521e3cb0182d31fd6a4691809cc0
* https://github.com/ethereum/go-ethereum/pull/30078/commits/bb182a64298f9a08023598da526bec85ae98ad19
* https://github.com/ethereum/go-ethereum/pull/30078/commits/6505e50d62ef554930f62cc325069665c03733f9

I'll add the updates, thanks